pom.xml

```diff
@@ -11,16 +11,16 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>spectrum</artifactId>
-    <version>1.6.1-SNAPSHOT</version>
+    <version>1.7.0</version>
     <name>Spectrum</name>
     <description>WebJar for Spectrum</description>
     <url>http://webjars.org</url>
 
     <developers>
         <developer>
-            <id>jamesward</id>
-            <name>James Ward</name>
-            <email>james@jamesward.com</email>
+            <id>yiyang00171</id>
```

Comment:
You can just add a new `<developer>` block.  You don't need to remove the existing one.